404.html

```diff
@@ -13,13 +13,12 @@
         <meta property="og:description" content="could you please stop linking to non-existant pages please">
     <!-- End of metatags.io content -->
     <!-- CSS -->
-      <!-- Main CSS -->
-        <!-- css gone -->
-      <!-- Overrides -->
+      <link rel="stylesheet" href="404.css">
     <!-- End of CSS -->
   </head>
   <body>
-    <h1>Hey!</h1>
+    <h1>Uh Oh</h1>
+    <img src="media/monkeys/telephonemonkey.gif"><src="media/monkeys/telephonemonkey.gif">
```

Comment:
why are there two tags lol

this contained the entire page after this in a `<src>` tag

Reply:
(maybe we should add some Alt Text too?)